staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -98,6 +98,10 @@ type KubeletConfiguration struct {
 	// Default: ""
 	// +optional
 	StaticPodPath string `json:"staticPodPath,omitempty"`
+	// podLogsRootDirectory is a custom root directory path kubelet will use to place pod's log files.
+	// Default: "/var/log/pods"
+	// +optional
+	PodLogsRootDirectory string `json:"podLogsRootDirectory,omitempty"`
```

Comment:
nit: can we make this consistent either with staticPodPath (e.g. `podLogsPath`) or volumePluginDir (e.g. `podLogsDir`)